Resources/Library/TTestBayesianOneSample.json

```diff
@@ -38,7 +38,9 @@
 			"format": "",
 			"name": "testValue",
 			"type": "Number",
-			"value": 0.0
+			"value": 0.0,
+			"max": 10000,
+			"min": -10000
```

Comment:
Did you also checked this for other data sets? The variables in debug?

Reply:
yes I did